### PR TITLE
[INLONG-5538][DataProxy] Optimize the Producer construction logic in TubeSink

### LINKED ISSUE #5538: [DataProxy] Optimize the Producer construction logic in TubeSink
### Description

Optimize the Producer construction logic in TubeSink

There are some places that need to be adjusted in the Producer construction logic in TubeSink, the main purpose of submitting this issue is to perform equivalent replacement, which is convenient for further optimization of the sink.

### InLong Component

InLong DataProxy

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by gosonzhang**
- Fixes #5538

### CODE REVIEW

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+                fbdTime = frozenTopicMap.get(topicName);
+                if (fbdTime == null) {
+                    AtomicLong tmpFbdTime = new AtomicLong();
+                    fbdTime = frozenTopicMap.putIfAbsent(topicName, tmpFbdTime);
+                    if (fbdTime == null) {
+                        fbdTime = tmpFbdTime;
+                    }
+                }
+                fbdTime.set(System.currentTimeMillis() + PUBLISH_FAILURE_WAIT);
+                logger.warn("Throw exception while publish topic="
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java:170]: Whether to print verbose stack trace log.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+    private static final long PUBLISH_FAILURE_WAIT = 60000L;
+    private final AtomicBoolean started = new AtomicBoolean(false);
+    private final String sinkName;
+    private final String clusterAddr;
+    private final MQClusterConfig clusterConfig;
+    private TubeMultiSessionFactory sessionFactory = null;
+    private final Map<String, MessageProducer> producerMap = new ConcurrentHashMap<>();
+    private MessageProducer lastProducer = null;
+    private final AtomicInteger lastPubTopicCnt = new AtomicInteger(0);
+    private static final ConcurrentHashMap<String, AtomicLong> frozenTopicMap
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java:54]: Suggest changing to UPPER_CASE for the `static final` variable.

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java (hunk 1 above)
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java:170]: No need, the error message of the TubeMQ has already fed back the cause of the abnormality

File: inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java (hunk 2 above)
Comment (gosonzhang) [reply inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/common/TubeProducerHolder.java]: DONE

**Review by healchow — APPROVED** [review]

**Review by luchunliang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by baomingyu — APPROVED** [review]
+1

**Pull request merged**